visualization/settings/templates.py

```diff
@@ -0,0 +1,59 @@
+import os
+from pathlib import Path
+
+path = os.path.abspath(Path(__file__).parent.parent.parent)
+header = f"export PYTHONPATH='{path}:$PYTHONPATH'"
+
+generate_solution_template = (
+    header
+    + """
+python3 nemo_skills/inference/generate_solutions.py \\
```

Comment:
Wouldn't it be easier to directly call main from generate_solutions and pass in config file? Then you don't need to map it like this if you have it as a config object in the code